src/libs/OptionsListUtils.js

```diff
@@ -681,7 +682,7 @@ function getOptions(reports, personalDetails, {
     if (login && (noOptions || noOptionsMatchExactly)
         && !isCurrentUser({login})
         && _.every(selectedOptions, option => option.login !== login)
-        && ((Str.isValidEmail(login) && !Str.isDomainEmail(login)) || Str.isValidPhone(login))
+        && ((Str.isValidEmail(login) && !Str.isDomainEmail(login)) || parsePhoneNumber(login).valid)
```

Comment:
Should we also use `possible` here? Or we're confident `valid` is enough?

Reply:
IMO we should only use `possible` (assuming valid is a subset of possible). We check the actual validity using twilio on the back end, so if we filter out just blatantly wrong numbers on the front end I think that's plenty, and I'd rather not take the chance of not letting a valid number through